internal/stdlib/syscall/exec_windows.go

```diff
@@ -392,29 +395,29 @@ func StartProcess(argv0 string, argv []string, attr *ProcAttr) (pid int, handle
 
 	// Do not accidentally inherit more than these handles.
 	if len(fd) > 0 {
-		err = updateProcThreadAttribute(si.ProcThreadAttributeList, 0, _PROC_THREAD_ATTRIBUTE_HANDLE_LIST, unsafe.Pointer(&fd[0]), uintptr(len(fd))*unsafe.Sizeof(fd[0]), nil, nil)
+		err = si.ProcThreadAttributeList.Update(windows.PROC_THREAD_ATTRIBUTE_HANDLE_LIST, 0, unsafe.Pointer(&fd[0]), uintptr(len(fd))*unsafe.Sizeof(fd[0]), nil, nil)
```

Comment:
same question here as for `Delete()`. it looks like `Update(...)` is doing more than `updateProcThreadAttribute`

Reply:
Same as the above, seems fine from looking.